src/docs/examples/recaptcha2.html

```diff
@@ -180,7 +180,7 @@ <h2>COVID-19</h2>
 
 
 
-<header id="qg-site-header" class="qg-site-header sticky">
+role="banner">
```

Comment:
I think the header was accidently replaced here?

Reply:
Yes. Thanks Ross. I'll fix it shortly.